common/pdf-jpg-export-js/PdfExportJS/PdfExportJS.csproj

```
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Telerik.UI.for.Blazor" Version="2.16.0" />
```

You can add the following package, so that libman dependencies are automatically downloaded upon build. If not - add a note in the ReadMe that the user should manually download them before running the project.

```suggestion
<PackageReference Include="Telerik.UI.for.Blazor" Version="2.16.0" />
<PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="2.1.76" />
```